katello/katello.spec

```diff
@@ -142,16 +142,17 @@ Useful utilities for debug info collecting
 %files debug
 %{_datadir}/foreman/script/foreman-debug.d/katello-debug.sh
 
-%package capsule
+%package foreman-proxy-content
```

Comment:
:+1: Looks good to me - this final package though will be named `katello-foreman-proxy-content` - is that what you want @ehelms?

If we want it w/out the katello prefix, use `%package -n foreman-proxy-content` (see https://docs.fedoraproject.org/en-US/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch10s04.html)